.gitignore

```diff
@@ -85,13 +85,13 @@ config/deploy/kustomization.yaml
 /.run/
 config/deploy/kubernetes/kubernetes.yaml
 config/deploy/kubernetes/kubernetes-csidriver.yaml
-config/deploy/kubernetes/kubernetes-others.yaml
+config/deploy/kubernetes/kubernetes-companions.yaml
```

Comment:
`companions` doesn't mean anything. If you don't like `others`, maybe `clusterwide`?